test/fixtures/rest/app/apis/users/posts/answer/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+exports.index = function* () {
```

Comment:
@popomore You are absolutely right. It is the idea. The PR update should reflect exactly the concept that you listed above. Code updated.